### PR TITLE
Fix throttling.init_policy test to accommodate new values

### PR DESCRIPTION
**Pull request by nabeelmmd**
Signed-off-by: Nabeel M Mohamed <nmeeramohide@micron.com>

## Issues Addressed
NFSE-4065

## Verified checks?
Have the checks completed?
- [x] Compile with latest HSE master
- [x] All new and existing unit and functional tests pass
- [] Code format check passes
- [x] All Commits are signed off

### CODE REVIEW

File: tests/test_kvdb.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ def test_sync(kvdb: hse.Kvdb):
 
 
 def test_param(kvdb: hse.Kvdb):
-    assert kvdb.param("throttling.init_policy") == '"default"'
+    assert (kvdb.param("throttling.init_policy") == '"heavy"' or
```
Comment (tristan957) [inline tests/test_kvdb.py:34]: What are your thoughts on presenting the configured `auto` back to the user?

Comment (tristan957) [reply tests/test_kvdb.py:34]: Not even sure how feasible that is.

Comment (tristan957) [reply tests/test_kvdb.py:34]: Makes sense to me.

**Comment by tristan957** [comment]
Nabeel can you rebase this PR on master? Big changes regarding testing went in.

File: tests/test_kvdb.py (hunk 1 above)
Comment (nabeelmmd) [reply tests/test_kvdb.py:34]: Agreed and yes it returns what's in effect...

**Review by tristan957 — APPROVED** [review]

**Pull request merged**